### ISSUE TITLE
Same Network

### ISSUE THREAD

**Issue by flaminbeast999** [loic2665/HeartRateToWeb#15]
Hi I got a problem. So How would I get this to work if I'm hardline to my router... I would unplug it but my computer is in a basement. Any solutions?

**Comment by vyazikov** [comment]
hardline? you need Wireless connection to your watch on phone in any case.
Does not matter where your PC is located, just must be at the same network.

**Comment by flaminbeast999** [comment]
Yeah thats the issue so my watch and phone are using the wireless. It's just that my desktop isn't finding the connection

**Comment by vyazikov** [comment]
check firewall. and the second quyestion -are you sure that python script executes without errors?

**Comment by flaminbeast999** [comment]
I was trying to use the csharp version

**Comment by loic2665** [comment]
Hello sorry for the late answer... well usually the problem is the firewall, try to add to rule to it. You can can either connect the watch to the wifi without the phone, or use bluetooth on your watch to the phone which is connected to your Wi-Fi.

**Comment by SooiDP** [comment]
I'm bumping into the same issue, is it possible to do this while your pc is connected through LAN and not wireless?

**Comment by loic2665** [comment]
Well, mine is only on wireless network so, it should work. What's your current network configuration ? Are devices alloew to communicate to each others ? Is your firewall blocking the connection ?

**Comment by SooiDP** [comment]
I resolved it by just using ngrok to forward it to external networks, couldn't be bothered with the pain of network debugging  😅

**Comment by loic2665** [comment]
But, but... the principle of the local connection.. xD 

No problem, if it works as you want... :)
